Modules/_cursesmodule.c

```diff
@@ -3916,13 +3927,18 @@ _curses_qiflush_impl(PyObject *module, int flag)
  * and _curses.COLS */
 #if defined(HAVE_CURSES_RESIZETERM) || defined(HAVE_CURSES_RESIZE_TERM)
 static int
-update_lines_cols(void)
+update_lines_cols(PyObject *module)
 {
     PyObject *o;
     PyObject *m = PyImport_ImportModuleNoBlock("curses");
```

Comment:
Why is the module still imported if you get it as an argument?

Reply:
@vstinner I didn't even notice this.  I added it as an argument for the PyModule_GetDict call